vermin/arguments.py

```diff
@@ -63,10 +63,9 @@ def print_usage(full=False):
       print("  --target=V | -t=V\n"
             "        Target version that files must abide by. Can be specified once or twice.\n"
             "        A '-' can be appended to match target version or smaller, like '-t=3.5-'.\n"
-            "        If not met Vermin will exit with code 1. Note that the amount of target\n"
-            "        versions must match the amount of minimum required versions detected.\n"
-            "        However, if used in conjunction with --violations, and no rules are\n"
-            "        triggered, it will exit with code 0.\n")
```

Comment:
Please don't remove this part about:
> However, if used in conjunction with --violations, and no rules are triggered, it will exit with code 0.

Add it again after the change you made beneath. So the "However, [...]" is after "[...] 2, you can just specify one target for Python 3.\n".